include/crpropa/module/EMPairProduction.h

```diff
@@ -35,12 +36,14 @@ class EMPairProduction: public Module {
 	EMPairProduction(
 		PhotonField photonField = CMB, //!< target photon background
 		bool haveElectrons = false,    //!< switch to create secondary electron pair
+		double thinning = 0,           //!< weighted sampling of secondaries
```

Comment:
Could we add a comment here that `thinning=0` means basically no thinning?
And in all other modules, too.

Reply:
Good idea! I've also added a better description of the thinning to the comments in the header files.